chpl/chpl-service/src/test/java/gov/healthit/chpl/app/chartdata/NonconformityTypeChartDataAppTest.java

```diff
@@ -0,0 +1,71 @@
+package gov.healthit.chpl.app.chartdata;
+
+import gov.healthit.chpl.auth.permission.GrantedPermission;
+import gov.healthit.chpl.auth.user.JWTAuthenticatedUser;
+import gov.healthit.chpl.dao.statistics.NonconformityTypeStatisticsDAO;
+import gov.healthit.chpl.dao.statistics.SurveillanceStatisticsDAO;
+import gov.healthit.chpl.dto.NonconformityTypeStatisticsDTO;
+
+import java.util.List;
+
+import junit.framework.TestCase;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+import org.springframework.test.context.support.DirtiesContextTestExecutionListener;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.github.springtestdbunit.DbUnitTestExecutionListener;
+import com.github.springtestdbunit.annotation.DatabaseSetup;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = { gov.healthit.chpl.CHPLTestConfig.class })
+@TestExecutionListeners({ DependencyInjectionTestExecutionListener.class,
+    DirtiesContextTestExecutionListener.class,
+    TransactionalTestExecutionListener.class,
+    DbUnitTestExecutionListener.class })
+@DatabaseSetup("classpath:data/testData.xml")
+public class NonconformityTypeChartDataAppTest extends TestCase {
+
+    @Autowired
+    private SurveillanceStatisticsDAO statisticsDAO;
+    
+    @Autowired 
+    private NonconformityTypeStatisticsDAO nonconformStatDAO;
+    
+    private static JWTAuthenticatedUser adminUser;
+	
+	@BeforeClass
+	public static void setUpClass() throws Exception {
+		adminUser = new JWTAuthenticatedUser();
+		adminUser.setFirstName("Administrator");
+		adminUser.setId(-2L);
+		adminUser.setLastName("Administrator");
+		adminUser.setSubjectName("admin");
+		adminUser.getPermissions().add(new GrantedPermission("ROLE_ADMIN"));
+	}
+
+    @Test
+    @Transactional
+    public void buildNonconformityTypeStatistics() {
```

Comment:
This isn't testing anything in the chartdata app, right? Just the DAO methods? If so, should be in a different test file